docs/asciidoc/faq.adoc

```diff
@@ -136,6 +136,8 @@ Issues related to TCP keep-alive configuration on various load balancers were re
 ** limit for buffering data in memory
 ** multipart exceeds the max file size limit
 ** bad request
+* Check the server.tomcat.max-keep-alive-requests property.
+** Some versions of spring default to 100, instead of unlimited.
```

Comment:
`server.tomcat.max-keep-alive-requests` value by default is **100** and is not changed


```suggestion

```